ios/FluentUI/AvatarGroup/AvatarGroup.swift

```diff
@@ -205,15 +205,13 @@ public struct AvatarGroup: View {
 
                 let ringPaddingInterspace = nextAvatarHasRing ? interspace - (ringOffset + ringOuterGap) : interspace - ringOffset
                 let noRingPaddingInterspace = nextAvatarHasRing ? interspace - ringOuterGap : interspace
-                let rtlRingPaddingInterspace = (nextAvatarHasRing ? -x - ringOuterGap : -x + ringOffset)
-                let rtlNoRingPaddingInterspace = (nextAvatarHasRing ? -x - ringOffset - ringOuterGap : -x)
                 let stackPadding = (currentAvatarHasRing ? ringPaddingInterspace : noRingPaddingInterspace)
 
+                let cutoutSize = isLastDisplayed ? (ringOuterGap * 2) + imageSize : nextAvatarSize
                 let xPosition = currentAvatarHasRing ? x - ringOuterGap - ringOuterGap : x - ringOuterGap
```

Comment:
to be consistent with your changes in line 210, should it be ringOuterGap *2?

Reply:
also for readability be consistent when you add -(currentavatarhasring? ringoutergap : 0 )?

Reply:
I was trying to keep the changes for this PR minimal and come back with another PR to update the other lines of the body, but I can update this one now, especially if I update it with rdeassis' suggestion.